### PR TITLE
Add description for TiKV Ready handled panel

### PR DESCRIPTION
**Pull request by tabokie**
https://asktug.com/t/topic/123305

### What is changed, added or deleted? (Required)

Add counter type description for TiKV "Ready handled" panel

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch
  - For v2.1, "v3.0.0 后废弃" can be removed
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by BusyJay — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -100,6 +100,13 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 ## Raft process
 
 - Ready handled：Raft 中不同 ready 类型的 ops
+  - count: 批量处理 ready 的 ops
+  - has_ready_region: 获得 ready 的 region 的 ops
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:104]:
```suggestion
  - has_ready_region: 获得 ready 的 Region 的 ops
```

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -100,6 +100,13 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+  - pending_region: 被检查是否获得 ready 的 region 的 ops，v3.0.0 后废弃
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:105]: 这一项是否还存在于面板中呢？

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -100,6 +100,13 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+  - message: ready 内待发送 message 的 ops
+  - append: ready 内 Raft log entry 的 ops
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:107]: 请问一下，这里的 ready 是一个术语名词吗？

File: grafana-tikv-dashboard.md (hunk 2 above)
Comment (tabokie) [reply grafana-tikv-dashboard.md:105]: 存在

File: grafana-tikv-dashboard.md (hunk 3 above)
Comment (tabokie) [reply grafana-tikv-dashboard.md:107]: 对

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -100,6 +100,13 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 ## Raft process
 
 - Ready handled：Raft 中不同 ready 类型的 ops
+  - count: 批量处理 ready 的 ops
+  - has_ready_region: 获得 ready 的 Region 的 ops
+  - pending_region: 被检查是否获得 ready 的 Region 的 ops，v3.0.0 后废弃
+  - message: ready 内待发送 message 的 ops
+  - append: ready 内 Raft log entry 的 ops
+  - commit: ready 内 committed Raft log entry 的 ops
+  - snapshot: 携带 snapshot 的 ready 的 ops
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:109]:
```suggestion
    - count: 批量处理 ready 的 ops
    - has_ready_region: 获得 ready 的 Region 的 ops
    - pending_region: 被检查是否获得 ready 的 Region 的 ops，v3.0.0 后废弃
    - message: ready 内待发送 message 的 ops
    - append: ready 内 Raft log entry 的 ops
    - commit: ready 内 committed Raft log entry 的 ops
    - snapshot: 携带 snapshot 的 ready 的 ops
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**